ipatests/prci_definitions/gating.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
         template: &ci-ipa-4-10-latest
-          name: freeipa/ci-ipa-4-10-f37
+          name: freeipa/ci-ipa-4-10-f38
           version: 0.0.2
```

Comment:
just to double check, is ci-ipa-4-10-f38 using the same version (0.0.2)?

Reply:
@f-trivino 
yes, you can check the latst version here: https://app.vagrantup.com/freeipa/boxes/ci-ipa-4-10-f38 and it is 0.0.2 as of today

Reply:
I got confused from https://github.com/freeipa/freeipa/pull/6825/commits/4f7011d1ab442e6dcf7fae8dcc42e79bdad6b47e

where I see 0.0.1

but yep, there is another version.